meilisearch-http/src/routes/tasks.rs

```diff
@@ -462,3 +536,126 @@ fn filter_out_inaccessible_indexes_from_query<const ACTION: u8>(
+            let json = r#" { "afterEnqueuedAt": "2021" } "#;
+            let err = serde_json::from_str::<TaskDeletionQuery>(json).unwrap_err();
+            snapshot!(format!("{err}"), @"could not parse an rfc3339- or iso8601-formatted date at line 1 column 30");
+        }
+        {
+            let json = r#" { "afterEnqueuedAt": "2021-12" } "#;
+            let err = serde_json::from_str::<TaskDeletionQuery>(json).unwrap_err();
+            snapshot!(format!("{err}"), @"could not parse an rfc3339- or iso8601-formatted date at line 1 column 33");
+        }
+        {
+            let json = r#" { "afterEnqueuedAt": "2021-12-03" } "#;
+            let query = serde_json::from_str::<TaskDeletionQuery>(json).unwrap();
+            snapshot!(format!("{:?}", query.dates.after_enqueued_at.unwrap()), @"2021-12-03 0:00:00.0 +00:00:00");
+        }
+        {
+            let json = r#" { "afterEnqueuedAt": "2021-12-03T23" } "#;
+            let err = serde_json::from_str::<TaskDeletionQuery>(json).unwrap_err();
+            snapshot!(format!("{err}"), @"could not parse an rfc3339- or iso8601-formatted date at line 1 column 39");
+        }
+        {
+            let json = r#" { "afterEnqueuedAt": "2021-12-03T23:45" } "#;
+            let query = serde_json::from_str::<TaskDeletionQuery>(json).unwrap();
+            snapshot!(format!("{:?}", query.dates.after_enqueued_at.unwrap()), @"2021-12-03 23:45:00.0 +00:00:00");
+        }
+        {
+            let json = r#" { "afterEnqueuedAt": "2021-12-03T23:45:23" } "#;
+            let query = serde_json::from_str::<TaskDeletionQuery>(json).unwrap();
+            snapshot!(format!("{:?}", query.dates.after_enqueued_at.unwrap()), @"2021-12-03 23:45:23.0 +00:00:00");
+        }
+        {
+            let json = r#" { "afterEnqueuedAt": "2021-12-03T23:45:23 +01:00" } "#;
+            let err = serde_json::from_str::<TaskDeletionQuery>(json).unwrap_err();
+            snapshot!(format!("{err}"), @"could not parse an rfc3339- or iso8601-formatted date at line 1 column 52");
+        }
+        {
+            let json = r#" { "afterEnqueuedAt": "2021-12-03T23:45:23+01:00" } "#;
+            let query = serde_json::from_str::<TaskDeletionQuery>(json).unwrap();
+            snapshot!(format!("{:?}", query.dates.after_enqueued_at.unwrap()), @"2021-12-03 23:45:23.0 +01:00:00");
+        }
+        {
+            let json = r#" { "afterEnqueuedAt": "1997-11-12T09:55:06.000000000-06:00" } "#;
+            let query = serde_json::from_str::<TaskDeletionQuery>(json).unwrap();
+            snapshot!(format!("{:?}", query.dates.after_enqueued_at.unwrap()), @"1997-11-12 9:55:06.0 -06:00:00");
+        }
+        {
+            let json = r#" { "afterEnqueuedAt": "1997-11-12T09:55:06.000000000Z" } "#;
+            let query = serde_json::from_str::<TaskDeletionQuery>(json).unwrap();
+            snapshot!(format!("{:?}", query.dates.after_enqueued_at.unwrap()), @"1997-11-12 9:55:06.0 +00:00:00");
+        }
+    }
+}
```

Comment:
Last issue, the date needs to always be exclusive

Reply:
done, I hope :)